rs/cli/src/discourse_client.rs

```diff
@@ -149,32 +179,54 @@ impl DiscourseClientImp {
             update_id: None,
         }))
     }
+
+    fn request_from_user_post(&self, err: anyhow::Error, body: String, topic_url: String) -> anyhow::Result<Option<DiscourseResponse>> {
+        warn!("Received error: {:?}", err);
+        warn!("Please create a post in topic {} with the following content", topic_url);
+        println!("{}", body);
+        let forum_post_link = dialoguer::Input::<String>::new()
+            .with_prompt("Forum post link")
+            .allow_empty(true)
+            .interact()?;
+        Ok(Some(DiscourseResponse {
+            url: forum_post_link,
+            update_id: None,
+        }))
+    }
 }
 
 const GOVERNANCE_TOPIC: &str = "Governance";
 const SUBNET_MANAGEMENT_TAG: &str = "Subnet-management";
```

Comment:
We need one more tag here, it was mentioned in one of the threads somewhere.

Reply:
I believe we cannot tag specific posts in a topic. The topics that exist are already properly tagged.

Reply:
These tags seem to be still used for authorized subnets proposals.
I've found the relevant doc in the meantime: https://docs.google.com/document/d/1Aar4w0JnUZTfXIaBp4PHvK7BP4G4s5IE6JxkuSfbycs/edit?pli=1&tab=t.0#heading=h.yd166uqdjfui

We should also add the subcategory "NNS proposal discussions" to these topics.

Reply:
Should that be a part of this PR? I can add it but seems like it should redone in another PR?

Reply:
Sure, another PR sounds good.